### PR TITLE
fixes header issue

### PR DESCRIPTION
**Pull request by abbyachau**

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -22,11 +21,9 @@
   - [Broker Errors](#broker-errors)
   - [Orphans](#orphans)
 
-<div id="changes"/> 
-## Changes ##
+## <div id="changes"/> Changes
```
Comment (duglin) [inline spec.md]: doesn't md add the anchors automatically for us?

Comment (abbyachau) [inline spec.md]: @duglin This pr attempts to fix the styling for all the headers in the current spec.md doc. Thanks, pls let me know if you want any further clarification.

Comment (duglin) [inline spec.md]: my point was that I don't think you need to manually add the "div" here.   In fact, this this "div" in there it gets an anchor named "-changes" which might throw off tooling that assumes normal github md processing.

Comment (angarg12) [inline spec.md]: I agree with @duglin, we should use the md anchors instead of divs.

Comment (abbyachau) [inline spec.md]: @duglin and @angarg12 thanks both. changes made per your feedback - this last commit includes the removal of the divs and minor changes to make the markdown anchors work properly. if you spot anything else, pls don't hesitate to get in touch. thanks again.

**Comment by angarg12** [comment]
Thanks a lot, unless I missed something it LGTM

**Comment by vaikas-google** [comment]
LGTM

**Comment by jmrodri** [comment]
LGTM what's required to merge this? the doc is annoying to read without these changes.

**Comment by duglin** [comment]
LGTM

@mattmcneeney in your PR I think you've added some div's - you may want to remove those to align with this change.

**Comment by mattmcneeney** [comment]
Thanks for the heads up @duglin. Have removed added divs in the branches I've been working on.

**Comment by pmorie** [comment]
LGTM

**Pull request merged**